app/src/main/java/com/nervousfish/nervousfish/modules/database/GsonDatabaseAdapter.java

```diff
@@ -48,13 +51,15 @@
 
     private final String contactsPath;
     private final String profilesPath;
+    private final IServiceLocator serviceLocator;
 
     /**
      * Prevents construction from outside the class.
      *
      * @param serviceLocator Can be used to get access to other modules
      */
     private GsonDatabaseAdapter(final IServiceLocator serviceLocator) {
+        this.serviceLocator = serviceLocator;
```

Comment:
The `serviceLocator` on `this` is never used, right? Also the log should come after this assignment :wink: